tremor-cli/src/test.rs

```diff
@@ -178,7 +183,33 @@ async fn run_integration(
         status::tags(&tags, Some(&matched), Some(&config.excludes))?;
 
         // Run integration tests
-        let test_report = process::run_process("integration", base, root, &tags).await?;
+        let test_report = match process::run_process("integration", base, root, &tags)
```

Comment:
could we make this a command-line argument instead of a hard-coded kill? I worry this will bite us at one point. Something like `tremor test --timeout-secs 300` ...